src/go/configgenerator/cluster_generator.go

```diff
@@ -205,6 +225,26 @@ func makeBackendCluster(opt *options.ConfigGeneratorOptions, brc *sc.BackendRout
 		LoadAssignment:       util.CreateLoadAssignment(brc.Hostname, brc.Port),
 	}
 
+	if opt.DnsResolverAddress != "" {
+		host, port, err := parseDnsResolverAddress(opt.DnsResolverAddress)
+		if err != nil {
+			return nil, fmt.Errorf("fail to parse dnsResolverAddress: %v", err)
+		}
+
+		c.DnsResolvers = []*corepb.Address{
```

Comment:
It seems that Envoy support multiple resolvers.   Should we support multiple?

My thought:  Most likely, users only need to use one.  it will be very very rare need to have 2.  
But since it is easy to support multiple,  why don't we just support it?
my concern is if we design this flag to only support one,  later if we want to support multiple with the same flag, it may break, not backward compatible.

Reply:
Done